FirebaseRemoteConfig/Sources/RCNConfigExperiment.m

```diff
@@ -1,202 +0,0 @@
-/*
- * Copyright 2019 Google
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#import "FirebaseRemoteConfig/Sources/RCNConfigExperiment.h"
-
-#import "FirebaseABTesting/Sources/Private/FirebaseABTestingInternal.h"
-#import "FirebaseCore/Extension/FirebaseCoreInternal.h"
-#import "FirebaseRemoteConfig/Sources/RCNConfigDefines.h"
-
-#import "FirebaseRemoteConfig/FirebaseRemoteConfig-Swift.h"
-
-static NSString *const kExperimentMetadataKeyLastStartTime = @"last_experiment_start_time";
-
-static NSString *const kServiceOrigin = @"frc";
-static NSString *const kMethodNameLatestStartTime =
-    @"latestExperimentStartTimestampBetweenTimestamp:andPayloads:";
-
-@interface RCNConfigExperiment ()
-@property(nonatomic, strong)
-    NSMutableArray<NSData *> *experimentPayloads;  ///< Experiment payloads.
-@property(nonatomic, strong)
-    NSMutableDictionary<NSString *, id> *experimentMetadata;  ///< Experiment metadata
-@property(nonatomic, strong)
-    NSMutableArray<NSData *> *activeExperimentPayloads;      ///< Activated experiment payloads.
-@property(nonatomic, strong) RCNConfigDBManager *DBManager;  ///< Database Manager.
-@property(nonatomic, strong) FIRExperimentController *experimentController;
-@property(nonatomic, strong) NSDateFormatter *experimentStartTimeDateFormatter;
-@end
-
-@implementation RCNConfigExperiment
-/// Designated initializer
-- (instancetype)initWithDBManager:(RCNConfigDBManager *)DBManager
-             experimentController:(FIRExperimentController *)controller {
-  self = [super init];
-  if (self) {
-    _experimentPayloads = [[NSMutableArray alloc] init];
-    _experimentMetadata = [[NSMutableDictionary alloc] init];
-    _activeExperimentPayloads = [[NSMutableArray alloc] init];
-    _experimentStartTimeDateFormatter = [[NSDateFormatter alloc] init];
-    [_experimentStartTimeDateFormatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"];
-    [_experimentStartTimeDateFormatter setTimeZone:[NSTimeZone timeZoneForSecondsFromGMT:0]];
-    // Locale needs to be hardcoded. See
-    // https://developer.apple.com/library/ios/#qa/qa1480/_index.html for more details.
-    [_experimentStartTimeDateFormatter
-        setLocale:[[NSLocale alloc] initWithLocaleIdentifier:@"en_US_POSIX"]];
-    [_experimentStartTimeDateFormatter setTimeZone:[NSTimeZone timeZoneWithAbbreviation:@"UTC"]];
```

Comment:
The `timeZone` was set twice, likely unintentionally. In the Swift implementation, I set it to UTC. A quick search says 0 seconds from GMT is practically the same as UTC.